app/models/internal/impl/DefaultReport.java

```diff
@@ -85,13 +89,50 @@ public ReportSource getSource() {
         return CompletableFuture.completedFuture(null);
     }
 
+    @Override
+    public String toString() {
+        final List<UUID> recipientIds =
```

Comment:
Is there a reason to custom serialize this versus relying on `List`'s `toString` and to do it?

Reply:
ah. I had this in while running the tests, it was a little bit easier to spot when using the hamcrest matchers but I did intend to revert that.